roofit/roofit/src/RooBukinPdf.cxx

```diff
@@ -16,24 +16,26 @@
 /** \class RooBukinPdf
     \ingroup Roofit
 
-The RooBukinPdf implements the NovosibirskA function. For the parameters, see
+RooBukinPdf implements the NovosibirskA function. For the parameters, see
 RooBukinPdf().
 
 Credits:
 May 26, 2003.
-A. Bukin, Budker INP, Novosibirsk
+A.Bukin, Budker INP, Novosibirsk
 
-\image html RooBukin.png
+http://www.slac.stanford.edu/BFROOT/www/Organization/CollabMtgs/2003/detJuly2003/Tues3a/bukin.ps
 **/
```

Comment:
The `\image html` should not be removed. Doxygen automatically adds the image to our website.

Reply:
I don't think I made these changes. Maybe i downloaded code from root repository